drivers/SmartThings/zwave-siren/profiles/multifunctional-siren.yml

```diff
@@ -12,10 +18,13 @@ components:
     version: 1
   - id: temperatureMeasurement
     version: 1
+    config:
+      values:
+        - key: "temperature.value"
+          range:
+            - -20
+            - 100
```

Comment:
looks like you wanted it in brackets: https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/pull/217/files

tagging in @ctowns since he's pretty familiar here

Reply:
Thanks! I tried several ways and the first one was closest: `range: "[ -20, 100 ]".` I'll just remove quotation marks.